docs/project/2024/your_curs.upb.ro_ana_maria.comeaga/your_curs_upb_ro_id.md

```diff
@@ -0,0 +1,109 @@
+| [Micro USB Breadboard 5V Power Supply Module] | Power Supply | [1.50 RON](https://cleste.ro/modul-alimentare-5v-micro-usb.html?utm_medium=GoogleAds&utm_campaign=&utm_source=&gad_source=1&gclid=Cj0KCQjwn7mwBhCiARIsAGoxjaI92As6Cx0y3WUm-b6heH19tuHWJqejoDrJV4l1LMAQLVLf7T4gvMsaAvLPEALw_wcB ) |
+| [5m 30AWG Silicone wires] | Connections | [19.50 RON](https://www.emag.ro/cablu-siliconic-ultraflexibil-14awg-bifilar-paralel-rosu-negru-cp14awg/pd/D1F6NPYBM/?cmpid=101180&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Iluminat_and_electrice&utm_content=76376892625&gad_source=1&gclid=Cj0KCQjwn7mwBhCiARIsAGoxjaLcDtf57lmj_iKgRz8embK0GAJY_Yc9ChU55L_7D7wNrAMA6JexHKgaAmeeEALw_wcB) |
+| [Perforated board] | Board | [75.42 RON](https://www.amazon.com/Double-Sided-Universal-Packs-12x18/dp/B08WJBX98F/ref=sr_1_2?crid=1389XD9ISH6EM&dib=eyJ2IjoiMSJ9.WsT-bgUR7rApIplnTIF8yABGZYgww0JAoqa8-54l1TNR355lFtbswumX6EjLYdIOuB_SDprsUzDVMyCV5Sf3ndWkgS4XmhOP8HSoGdqktLGlJqg9Kh14NEdeJAT_JswO9_-zVhuHUjgOttFwD51niWmp1Ki8zQtB2HBQBP7-cPFg__v6N7uQ8rY8OwfxeY_ksj0Y3gb3BFI9VtQ0sBsLcDSK-MZGQrwlJe_i-XfpMM4.4ACAhff_DsTRrHhx7lc8inTj4yOSTxPrQVeq5I2V4hs&dib_tag=se&keywords=12%2Bx%2B18%2Bcm%2BUniversal%2BPCB%2BPrototype%2BBoard%2BDouble-Side&qid=1712244151&sprefix=12%2Bx%2B18%2Bcm%2Buniversal%2Bpcb%2Bprototype%2Bboard%2Bdouble-side%2Caps%2C329&sr=8-2&th=1) |
+| [Watering Pump] | Watering the system | [26.13 RON](https://www.emag.ro/pompa-de-apa-kruger-meier-sturmkraft-220jk-x1-220l-h-3-8-x-3-7-x-2-8-cm-negru-5903900042987/pd/DTFJ05MBM/?X-Search-Id=7b217b870d8e7e7ff1d5&X-Product-Id=113031364&X-Search-Page=1&X-Search-Position=32&X-Section=search&X-MB=0&X-Search-Action=view) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+| [ads1x1x](https://github.com/vgasparyan/ads1256-rs) | Provides drivers for ADS1x1x ADCs | Used to interact with the ADS1115 ADC |
+| [std](https://github.com/rust-lang/rust/blob/master/library/std/src/lib.rs) | Standard library for Rust | Provides core functionality and data structures |
+| [embedded-hal](https://github.com/embassy-rs/embassy) | Hardware abstraction layer for embedded systems | Used for abstracting hardware interactions |
+| [rppal](https://github.com/golemparts/rppal) | Raspberry Pi Pico hardware interface library | Interface with Raspberry Pi Pico hardware components |
+| [lcd_2inch](https://github.com/zadi15/picoLCD) | Library for interfacing with 2-inch LCD displays | Initializing and controlling the LCD display |
+| [thread](https://github.com/rust-lang/rust/blob/master/library/std/src/thread/mod.rs) | Thread management library | Creating and managing threads |
```

Comment:
I think this is from the standar library and is not available for embedded systems.